staging/src/k8s.io/client-go/applyconfigurations/meta/v1/deleteoptions.go

```diff
@@ -36,7 +36,10 @@ type DeleteOptionsApplyConfiguration struct {
 // DeleteOptionsApplyConfiguration constructs an declarative configuration of the DeleteOptions type for use with
 // apply.
 func DeleteOptions() *DeleteOptionsApplyConfiguration {
```

Comment:
pre-existing, but apply configuration for delete options is weird, right? we can't actually submit this object to server-side apply

Reply:
I remember squinting at this one when I originally added apply configurations.  The apply configurations generator uses a reachability routine to figure out which types to include. It includes delete options because of https://github.com/kubernetes/kubernetes/blob/6a5b3da1d8748becdb26d8d8a387019a5627f2a7/staging/src/k8s.io/api/policy/v1/types.go#L171